adscompstat/tasks.py

```diff
@@ -453,3 +471,25 @@ def task_export_completeness_to_json():
                 )
         except Exception as err:
             logger.error("Unable to export completeness data to disk: %s" % err)
+
+
+@app.task(queue="get-logfiles")
```

Comment:
Did you mean to have this task as async? It seems fast, spinning off async tasks for the heavier parts, and this queue is already being used for another task.

Reply:
Keeping it async will let it process the unmatched, mismatched, and failed records simultaneously, in the same way that the other task using this queue will process each logfile simultaneously.  The two tasks will never run at the same time so I thought reusing the queue would be more efficient, but I can make a new queue specifically for this for clarity.